apps/client/src/app/components/admin-overview/admin-overview.html

```diff
@@ -30,19 +30,23 @@
               }
             </div>
           </div>
-          <div class="d-flex my-3">
-            <div class="w-50" i18n>User Signup</div>
-            <div class="w-50">
-              <mat-slide-toggle
-                color="primary"
-                hideIcon="true"
-                [checked]="
-                  info.globalPermissions.includes(permissions.createUserAccount)
-                "
-                (change)="onEnableUserSignupModeChange($event)"
-              />
+          @if (hasPermissionForAuthToken) {
```

Comment:
This should not be changed, we always need the possibility to disable new sign ups